content/en/boilerplates/untested-document.md

```diff
@@ -0,0 +1,5 @@
+---
+---
+{{< warning >}}
+This document has not been tested on the Istio 1.8 release and may contain bugs.
```

Comment:
Should this be a 1.9 statement instead of 1.8 (although we won't have tested on either)?

Reply:
Changed. I will do a manual cherry pick into 1.8 when this gets merged.